src/config/package-documentation/index.md

```diff
@@ -0,0 +1,16 @@
+# Package documentation
+
+The most common media for documentation in Void Linux are [manual
+pages](./man.md).
+
+Many packages contain documentation in other formats, like HTML. This
+documentation can usually be found in a `/usr/share/doc/<package>` directory.
+
+More extensive documentation may be split into separate `*-doc` packages, such
+as `julia-doc`. This is often the case for programming languages, databases and
+big software libraries.
+
+In addition to documentation provided by upstream projects, packages may also
+contain description of initial setup or usage specific to Void, provided by
+distro contributors. It will be located in
```

Comment:
My only nit is "distro" -> "distribution", which is more widespread in docs. Otherwise LGTM

Reply:
Done.